learn/data_backup/dumps.mdx

```diff
@@ -53,34 +61,32 @@ This command should return an object with detailed information about the dump op
 }
 ```
 
-The dump creation process is an asynchronous task that takes time proportional to the size of your dataset. All indexes of the current instance are exported along with their documents and settings and saved as a single `.dump` file.
+All indexes of the current instance are exported along with their documents and settings and saved as a single `.dump` file.
 
-After dump creation is finished—when `status` is `succeeded`—the dump file is added to the dump directory. By default, this folder is named `dumps` and can be found in the same directory as your Meilisearch binary. You can customize [this using the `--dump-dir` configuration option](/learn/configuration/instance_options#dump-directory). **If the dump directory does not already exist when the dump creation process is called, Meilisearch will create it.**
+Once the task `status` changes to `succeeded`, find the dump file in [the dump directory](/learn/configuration/instance_options#dump-directory). By default, this folder is named `dumps` and can be found in the same directory as your Meilisearch binary.
```

Comment:
Maybe at this point, we should also mention that all the tasks registered after the dump task **started processing** won’t be in the dump in the end?